docs/sql-ref-syntax-aux-resource-mgmt-list-jar.md

```diff
@@ -26,7 +26,8 @@ license: |
 ### Syntax
 
 ```sql
-LIST JAR
+LIST { JAR | JARS } file_name [ ... ]
```

Comment:
Thanks for this update. I think its better to backport this doc update into the other branches.